docs/handle-survey-results-access.md

```diff
@@ -0,0 +1,114 @@
+# Access Survey Results
+
+This help topic describes how you can access, modify, and display individual question values and full survey results.
+
+- [Access Individual Question Values](#access-individual-question-values)
+- [Access Full Survey Results](#access-full-survey-results)
+
+## Access Individual Question Values
+
+To access an individual question value, use the question's [`value`](/Documentation/Library?id=Question#value) property. You can assign a new value to it if you want to update the question value programmatically. Alternatively, you can call the Survey's [`getValue(questionName)`](/Documentation/Library?id=surveymodel#getValue) and [`setValue(questionName, newValue)`](/Documentation/Library?id=surveymodel#setValue) methods.
+
+The following code configures an `nps_score` question and uses the API members described above to access the question's value:
+
+```js
+import { Model } from "survey-core";
+
+const surveyJson = {
+  "elements": [{
+    "type": "rating",
+    "name": "nps_score",
+    "title": "On a scale of zero to ten, how likely are you to recommend our product to a friend or colleague?",
+    "rateMin": 0,
+    "rateMax": 10,
+  }]
+};
+const survey = new Model(surveyJson);
+const npsQuestion = survey.getQuestionByName("nps_score");
+
+console.log(npsQuestion.value);
+// ===== or =====
+// console.log(survey.getValue("nps_score"));
+
+npsQuestion.value = 10;
+// ===== or =====
+// console.log(survey.setValue("nps_score", 10));
+```
+
+If you need to access an individual question value before or right after it is changed, handle the Survey's [`onValueChanging`](/Documentation/Library?id=surveymodel#onValueChanging) or [`onValueChanged`](/Documentation/Library?id=surveymodel#onValueChanged) events:
+
+```js
+survey.onValueChanging.add((survey, { name, question, oldValue, value }) => {
+  console.log(`The ${name} question value is about to change from ${oldValue} to ${value}.`);
+  // You can redefine the `value` argument if you want to change the question value:
+  // value = myNewValue;
+});
+
+survey.onValueChanged.add((survey, { name, question, value }) => {
+  console.log(`The ${name} question value has changed to ${value}.`);
+});
+```
+
+## Access Full Survey Results
+
+To access full survey results, use the Survey's [`data`](/Documentation/Library?id=surveymodel#data) property. It is a JSON object that contains only answered questions and has the following structure:
+
+```js
+{
+  "question1Name": "question1Value",
+  "question2Name": "question2Value",
+  // ...
+  "questionNName": "questionNValue",
+}
+```
+
+Depending on the question type, question values can also be nested objects or arrays that contain objects or primitive values:
```

Comment:
What do you mean by "primitive values"? Simple values compared to objects?

```suggestion
Depending on the question type, question values can also be nested objects or arrays that contain objects or simple values:
```

Reply:
Primitive [is a proper term](https://developer.mozilla.org/en-US/docs/Glossary/Primitive) in JS.